src/posix.mak

```diff
@@ -22,9 +22,15 @@ INSTALL_DIR=../../install
 SYSCONFDIR=/etc
 PGO_DIR=$(abspath pgo)
 
-C=backend
-TK=tk
-ROOT=root
+D = ddmd
+
+C=$D/backend
+TK=$D/tk
+ROOT=$D/root
+
+GENERATED = ../generated
+G = $(GENERATED)/$(OS)$(MODEL)
+$(shell mkdir -p $G)
```

Comment:
> the path will be generated/dmd

Didn't we agree on `generated/linux/debug/64/dmd` ?
In the final build step you could create a symlink to the last built binary (e.g `generated/dmd` -> ` `$(abspath generated/linux/debug/64/dmd)`) if you really want to ...